scenario/features/sbe.feature

```diff
@@ -35,3 +35,40 @@ Feature: State-based endorsement
         And I set the transaction arguments to ["key-001"]
         And I invoke the transaction
         Then the response should be "value-001"
+
+    Scenario: Set SBE policy for Org1 AND Org3; state change successfully when not specifying endorsing orgs
+        When I prepare to submit a SetStateEndorsers transaction
+        And I set the transaction arguments to ["key-001", "Org1MSP", "Org3MSP"]
+        And I invoke the transaction
+        And I prepare to submit a ChangeState transaction
+        And I set the transaction arguments to ["key-001", "value-Org1-Org3"]
+        And I invoke the transaction
+        And I prepare to evaluate a GetState transaction
+        And I set the transaction arguments to ["key-001"]
+        And I invoke the transaction
```

Comment:
Shouldn't we use a verb like "evaluate" or "query" here instead of "invoke"?

Reply:
It's a neutral term because to evaluate a transaction we would use:

And I prepare to **evaluate** a GetState transaction
And I set the transaction arguments to ["key-001"]
And I invoke the transaction

To submit a transaction:

And I prepare to **submit** a GetState transaction
And I set the transaction arguments to ["key-001"]
And I invoke the transaction